client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss

```diff
@@ -176,8 +175,8 @@
 	letter-spacing: 0;
 	padding: 20px 32px;
 	// @TODO: work out hover-state animations
```

Comment:
Is this `@TODO` valid anymore?

Reply:
This styles will go away after a rebase. From now on we should stick to using/maintaining just [action-buttons](https://github.com/Automattic/wp-calypso/blob/master/client/landing/gutenboarding/components/action-buttons/index.tsx) generic components.